.structure-config

```diff
@@ -13,3 +13,10 @@ scripts/testing/
 dracut/.shellcheckrc
 docs/ci-status.rst
 )
+
+RPM_REBUILD_PATHS=(
+anaconda.spec.in
+dockerfile/anaconda-rpm
+.github/workflows/tests.yml
+branch-config.mk
+)
```

Comment:
I am not sure if this list has everything, but it seems like it could be a good start.

I considered also including Makefile, but only the top level one applies, because it starts the container with tests. Current pattern matching would take all makefiles.

It's trivial to make it so that only a top level Makefile can be specified - change
```diff
- if [[ "$iter_f" =~ "$rebuild_f" ]]; then
+ if [[ "./$iter_f" =~ "$rebuild_f" ]]; then
```
and then files can be specified relative to repo root as eg. `./Makefile`